assertion/function/analyzer.go

```diff
@@ -137,9 +137,9 @@ func run(pass *analysis.Pass) (result interface{}, _ error) {
 		// Construct config for analyzing the functions in this file. By default, enable all checks
 		// on NilAway itself.
 		functionConfig := assertiontree.FunctionConfig{}
-		if strings.HasPrefix(pass.Pkg.Path(), config.NilAwayPkgPathPrefix) {
-			functionConfig.StructInitCheckType = config.DepthOneFieldCheck
-			// TODO: enable anonymous function flag here when  is done
+		if strings.HasPrefix(pass.Pkg.Path(), config.NilAwayPkgPathPrefix) { //nolint:revive
+			// TODO: enable struct initialization flag.
```

Comment:
Why are we removing this now? Is it that we need to add it as another CLI flag for the stand alone mode? Is that hard to do?